container/containerd/handler.go

```diff
@@ -32,6 +33,9 @@ import (
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 )
 
+// TODO: if it is decided that this is the only format, this flag can be removed.
+var useK8sContainerNameFormat = flag.Bool("use_k8s_container_name_format", true, "Use the k8s format: k8s_<container-name>_<pod-name>_<namespace>_<pod-uid>_<restart-count>")
```

Comment:
please update https://github.com/google/cadvisor/blob/master/docs/runtime_options.md with this flag

Reply:
s/Use the k8s format: .../as the name and alias for the container